project_hr/models/hr_employee.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Tecnativa - Victor M.M. Torres
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    @api.model
+    def create(self, vals):
+        employee = super().create(vals)
+        if employee.category_ids:
+            tasks = self.env['project.task'].search([
```

Comment:
No need of searching involved tasks. `invalidate_cache` works at model level (it's `@api.model`).

Reply:
And what happens when you modify category_ids of an employee? Add a test to check if you need to invalidate cache as well.

Reply:
Think already have https://github.com/OCA/project/pull/568/files?file-filters%5B%5D=.py#diff-21c8ed16cfa23508bf379ef5143fdc4eR45

Reply:
No, do it a second time on same tests and it will be then the proper test, as first time there's no value cached.